### ISSUE TITLE
[gatsby-source-twitter] Not generating GraphQL nodes

### ISSUE THREAD

**Issue by dderooy** [gatsbyjs/gatsby#10889]
## Description
I am trying to use gatsby-source-twitter plugin to get tweets from a company Twitter profile and display them on the gatsby index page of the new company website. First I setup a [twitter developer account](https://developer.twitter.com) and created a twitter app on that site. Once I had the consumer keys I followed [these instructions](https://developer.twitter.com/en/docs/basics/authentication/guides/bearer-tokens.html) to get the bearer token. After installing the plugin in my gatsby dev workspace (npm installed plugin and followed setup in gatsby-config.js) I used the Twitter query format '@CompanyTwitterAccount'. 

When running 'gatsby develop' in terminal I noticed it said **"warning The gatsby-source-twitter plugin has generated no Gatsby nodes. Do you need it?"**. After the site finished loading I went to the graphQL tool and saw that indeed there was no node for 'allTweet' as described in the documentation. 

## Troubleshooting
I double checked the bearer token creation and the consumer API keys with no luck. I also checked the twitter query '@CompanyTwitterAccount' on the normal twitter site to make sure tweets show up.

### Environment
  System:
    OS: macOS 10.14.2
    CPU: (8) x64 Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 11.1.0 - /usr/local/bin/node
    Yarn: 1.12.3 - /usr/local/bin/yarn
    npm: 6.4.1 - /usr/local/bin/npm
  Browsers:
    Chrome: 71.0.3578.98
    Firefox: 64.0
    Safari: 12.0.2
  npmPackages:
    gatsby: ^2.0.53 => 2.0.53
    gatsby-image: ^2.0.25 => 2.0.25
    gatsby-mdx: ^0.2.0 => 0.2.0
    gatsby-plugin-eslint: ^2.0.1 => 2.0.1
    gatsby-plugin-offline: ^2.0.5 => 2.0.5
    gatsby-plugin-react-helmet: ^3.0.0 => 3.0.0
    gatsby-plugin-react-svg: ^2.0.0 => 2.0.0
    gatsby-plugin-sharp: ^2.0.17 => 2.0.17
    gatsby-plugin-sitemap: ^2.0.1 => 2.0.1
    gatsby-plugin-styled-components: ^3.0.0 => 3.0.0
    gatsby-plugin-webpack-size: ^0.0.2 => 0.0.2
    gatsby-source-filesystem: ^2.0.1 => 2.0.1
    gatsby-source-twitter: ^1.3.4 => 1.3.4
    gatsby-transformer-json: ^2.1.1 => 2.1.1
    gatsby-transformer-remark: ^2.1.1 => 2.1.1
    gatsby-transformer-sharp: ^2.1.1 => 2.1.1
  npmGlobalPackages:
    gatsby-cli: 2.4.4

**Comment by dderooy** [comment]
After experimenting more, I found that if I use other twitter queries such as "@Tech" then the nodes are generated. So the plugin is working fine then! I have to figure out why the twitter query for the company is not behaving as expected. Maybe it has some privacy setting set somewhere.

**Comment by MaxMcKinney** [comment]
I am also having this issue, my smaller company tweeter does not generate nodes, however using something like @apple works. Has there been anymore development?

**Comment by dderooy** [comment]
Yea I think it was some niche use of the Twitter API that requires a different query somehow. I think I was using the wrong endpoint even though it seemed like the right one? 

I ended up using [gatsby-plugin-twitter](https://www.gatsbyjs.org/packages/gatsby-plugin-twitter/) instead of the API source plugin. Then I used [Twitter Publish](https://publish.twitter.com/#) to embed the company timeline and set some parameters for number of tweets etc.  

Although this worked and was easy to implement, its the slowest part of the page load since the images are not optimized. I'm wondering if the source API will be faster since then gatsby images can be used. If you get the source plugin working let me know!
